apps/src/applab/dataBrowser/TableControls.jsx

```diff
@@ -51,9 +62,11 @@ const TableControls = React.createClass({
   render() {
     return (
       <div style={styles.container}>
-        <button className="btn" onClick={this.props.addColumn} style={dataStyles.button}>
-          Add column
-        </button>
+        <span style={styles.tableNameWrapper}>
+          <span style={styles.tableName}>
+            {this.props.tableName}&nbsp;
```

Comment:
This `&nbsp;` is unnecessary, since I no longer need a gap between the table name and the Add Column button now that the Add Column button has gone away. However, `&nbsp;` seems to be working fine in the "Back to data" links, so I've left them there.